index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Harmattan&family=Inter:wght@100&family=Poppins:wght@200&family=Roboto+Serif:wght@100&family=Roboto:ital,wght@0,400;1,100&display=swap"
+      rel="stylesheet"
+    />
+
+    <link rel="stylesheet" href="CSS/style.css" />
+    <title>My-Portfolio</title>
+  </head>
+  <body>
+    <section>
+      <header>
+        <nav>
+          <div class="mobile-nav">
+            <a href="#" class="name_logo">Gidiz Logo</a>
+            <img class="menu-bar" src="icons/menu.png" alt="Menu" />
+          </div>
+        </nav>
+      </header>
+
+      <div class="container">
+        <h1>Hey There. I’m Gideon</h1>
+        <h2>I’am a Software Developer</h2>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+
+        <div class="social">
+          <img src="icons/social/github.png" alt="github" />
+          <img src="icons/social/linkedin.png" alt="linkedin" />
+          <img src="icons/social/angellist.png" alt="angellist" />
+          <img src="icons/social/twitter.png" alt="twitter" />
+          <img src="icons/social/medium.png" alt="medium" />
+        </div>
+        <img class="side_img" src="icons/side_img.png" alt="" />
+      </div>
```

Comment:
[REQUIRED] the requirement said that you should implement two sections the first one is the header and good job for that 😎and the second one is the headline but in your code, I see that you use a  div element  to wrap your second section which is semantically not true check this [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) article about section element 

![image](https://user-images.githubusercontent.com/74744787/158694250-9d173108-9d02-400f-8d92-5cb8a53826dc.png)

- What you can  do : 
  - Please change the div element with the class container to a section element 
  - change the section element that contains all your code to the main element or to the div  element
- generally: 
  - If the contents represent the main content area of a document, use (main) element
  - If you are only using the element as a styling wrapper, use a (div)  A rule of thumb is that a (section) should logically appear in the outline of a document.